### DISCUSSION TITLE
how to remove edit capability from specific admin view?

### DISCUSSION THREAD

**Discussion by saman222** [joomengine/Joomla-Component-Builder#765] [Q&A]
hi
i have an admin view and i want only in this, remove edit record possibility  and not with joomla permission tab. i want admin use only can add or remove record. how to implement that with JCB GUI? is it possible? 
also it is enough if you give me your youtube link i can use that. i can't find anything for do this in the tutorials.
thank you

**Comment by Llewellynvdm** [comment]
There is no GUI way to do this far as I can remember. So you will need to do a custom code replacement of these buttons in the view.html.php file of the admin list view.
